modules/boot-api/src/main/java/org/mule/runtime/module/boot/api/MuleContainerLifecycleWrapper.java

```diff
@@ -20,9 +18,7 @@ public interface MuleContainerLifecycleWrapper {
    *
    * @return The {@link MuleContainerLifecycleWrapper} implementation.
    */
-  public static MuleContainerLifecycleWrapper getMuleContainerWrapper() {
-    return MuleContainerWrapperProvider.getMuleContainerWrapper();
-  }
+  public MuleContainerLifecycleWrapper getMuleContainerWrapper();
```

Comment:
This one must remain static, we are breaking backwards otherwise. Why not calling `MuleContainerLifecycleWrapperProvider#load` here and passing `MuleContainerLifecycleWrapper.class.getClassLoader()`